lib/tbd/ua.rb

```diff
@@ -712,20 +712,35 @@ def ua_summary(date = Time.now, argh = {})
           next unless edge.key?(:psi)
 
           loss = edge[:length] * edge[:psi]
-          type = edge[:type].to_s
-
-          case type
-          when /rimjoist/i     then bloc[:pro][:rimjoists] += loss
-          when /parapet/i      then bloc[:pro][:parapets ] += loss
-          when /roof/i         then bloc[:pro][:parapets ] += loss
-          when /fenestration/i then bloc[:pro][:trim     ] += loss
-          when /head/i         then bloc[:pro][:trim     ] += loss
-          when /sill/i         then bloc[:pro][:trim     ] += loss
-          when /jamb/i         then bloc[:pro][:trim     ] += loss
-          when /corner/i       then bloc[:pro][:corners  ] += loss
-          when /balcony/i      then bloc[:pro][:balconies] += loss
-          when /grade/i        then bloc[:pro][:grade    ] += loss
-          else                      bloc[:pro][:other    ] += loss
+          # type = edge[:type].to_s
+          type = edge[:type].to_s.downcase
+
+          if edge[:type].to_s.downcase.include?("balcony")
+            bloc[:pro][:balconies] += loss
+          elsif edge[:type].to_s.downcase.include?("door")
+            bloc[:pro][:trim     ] += loss
+          elsif edge[:type].to_s.downcase.include?("skylight")
```

Comment:
Change in approach to avoid double-counting (e.g. **:balconysill** vs **:doorsill**).